code/parachain/runtime/picasso/src/ibc.rs

```diff
@@ -104,6 +105,28 @@ impl core::str::FromStr for MemoMessage {
 	}
 }
 
+parameter_types! {
+	pub const GRANDPA: pallet_ibc::LightClientProtocol = pallet_ibc::LightClientProtocol::Grandpa;
+	pub const IbcTriePrefix : &'static [u8] = b"ibc/";
+}
+
+use pallet_ibc::ics20::Ics20RateLimiter;
+
+pub struct ConstantAny;
+
+impl Ics20RateLimiter for ConstantAny {
+	fn allow(
+		msg: &pallet_ibc::ics20::Ics20TransferMsg,
+		_flow_type: pallet_ibc::ics20::FlowType,
+	) -> Result<(), ()> {
+		// one DOT/PICA, so so for USDT not safe, but we do not yet do it
+		if msg.token.amount.as_u256() <= ::ibc::bigint::U256::from(10_u64.pow(12)) {
```

Comment:
where's this limit coming from?

Reply:
1 DOT/PICA

Reply:
12 decimals.

sure, will need to implement real stuff when have some full design. or split constant per channel (needs decimals, better not to do until assets merged, so in theory can do).

Reply:
we can change any value. so would prefer we have some RL pallet.